Lombiq.VueJs.Samples/Controllers/QrCardController.cs

```diff
@@ -0,0 +1,41 @@
+using Lombiq.VueJs.Samples.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.ContentManagement;
+using OrchardCore.Mvc.Core.Utilities;
+using System.Threading.Tasks;
+
+namespace Lombiq.VueJs.Samples.Controllers;
+
+public class QrCardController : Controller
+{
+    private readonly IContentManager _contentManager;
+
+    public QrCardController(IContentManager contentManager) =>
+        _contentManager = contentManager;
+
+    // Open this from under /Lombiq.VueJs.Samples/QrCard/Index.
+    public ActionResult Index()
+    {
+        var linkGenerator = HttpContext.RequestServices.GetRequiredService<LinkGenerator>();
```

Comment:
Looks like a lazy dependency to me, isn't it?

Reply:
Can you explain, please?

Reply:
Good question. I'll do that.

Reply:
I'm not sure I can use `Lazy<T>` here, but I'll try it.

Reply:
There is no `Lazy<LinkGenerator>` in the service container, but I added `LinkGenerator` as constructor dependency.

Reply:
If **Autofac** is used, `Lazy<T>` resolves automatically.

Reply:
😏 OK! Thanks for the insight.

Reply:
For MS DI we have an extension method in _Lombiq.HelpfulLibraries.Common_ that enables Lazy injection. [See docs here.](https://github.com/Lombiq/Helpful-Libraries/blob/80c5e6c9ac65a0e1c58cb4413ae12228edd59d35/Lombiq.HelpfulLibraries.Common/Docs/DependencyInjection.md#lazy-injection-support)

Reply:
Good to know, thank you.

Reply:
Wow! So much magic hidden away ✨